### ISSUE TITLE
named views?

### ISSUE THREAD

**Issue by jmlopez-rod** [ui-router/ng1-to-ng2#6]
does this project support named views like the base ui-reouter project does?  I cant seem to get a component injected into a ui-view with a name eg 

```
views: {
  '@home': {
    component: AppComponent
  }
}
```
works but 
```
views: {
  'viewName@home': {
    component: AppComponent
  }
}
```

won't

**Comment by christopherthielen** [comment]
confirmed, named views not working. I will make an effort to fix this.

**Comment by christopherthielen** [comment]
fixed in a2ce4a864bced4f39b933e36f457bdff16fc1d0a
